src/Entity/ClientName.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TeaEbook\Oauth2BundleExtended\Entity;
+
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+
+class ClientName
+{
+    /** @var int|null */
+    protected $id;
+
+    /** @var Client|null */
+    protected $client;
+
+    /** @var string|null */
+    protected $name;
+
+    public function __construct(Client $client, string $name = null)
```

Comment:
Does it actually make sense to create one without a name?

Reply:
Oh, so it is mandatory to have a row in `oauth2_client_name` even if the client does not have a name? I think that is what confused me, but I guess that makes it consistent with the name column being optional.

Reply:
I've just take the old code to move it here, i don't want to change this behavior

we have only one creation of this entity, and name is optionnal: https://github.com/TEA-ebook/oauth2-bundle-extended/blob/bundle/src/Command/CreateClientCommand.php#L189 (`$input->getOption('name')` is null if we don't add `--name=` in CLI)